modules/creating-a-drift-metric-using-cli.adoc

```diff
@@ -0,0 +1,63 @@
+:_module-type: PROCEDURE
+
+[id='creating-a-drift-metric-using-cli_{context}']
+= Creating a data drift metric by using the CLI
+
+[role='_abstract']
+You can use the OpenShift command-line interface (CLI) to create a data drift metric for a model.
+
+.Prerequisites
+
+* You are familiar with the specific data set schema and understand the names and meanings of the inputs and outputs.
+
+* Your OpenShift cluster administrator added you as a user to the {openshift-platform} cluster and has installed the TrustyAI service for the data science project where the models are deployed.
+
+* You set up TrustyAI for your data science project, as described in xref:setting-up-trustyai-for-your-project_monitor[Setting up TrustyAI for your project].
+
+.Procedure
+
+. In a terminal window, log in to the OpenShift cluster where {productname-short} is deployed.
++
+----
+oc login
```

Comment:
Is this login command sufficient, or do they need to log in with the token again? 

$ oc login --token=__<token>__ --server=__<openshift_cluster_url>__